locales/en/translations.json

```diff
@@ -337,7 +337,14 @@
        "shares" : "Shares",
        "servicesOrGoods" : "Services or goods",
        "anyOtherAsset" : "Any other asset",
-       "compositionErrorMessage" : "Select a tick box"
+       "compositionErrorMessage" : "Select a tick box",
+       "contributionSubTypes": {
```

Comment:
Rather than add this new structure with duplicated values, can't you just reference the existing key/values above?